app/app/settings.py

```diff
@@ -46,7 +46,6 @@
 # Application definition
 
 INSTALLED_APPS = [
-    'whitenoise.runserver_nostatic',
```

Comment:
How did you find out this was the problem ?

Reply:
Logging errors helped finding out statics were non findable.
_WhiteNoise_ is a third party app to manage statics in Django.
disabling WhiteNoise and going for native Django just works fine enough for our usage.